### ISSUE TITLE
Add world to the group even if the world doesn't exist yet

### ISSUE THREAD

**Issue by bobhenl** [Multiverse/Multiverse-Inventories#419]
### Feature request

**Feature description**
Hi there¨s probem with multiverse inventories... when I wanted to use PlayerWorldsPRO and multiverse inventories... I cannot add non existing worlds into group... but when I execute the command to add world to the group on creation, the world isn't yet created and isn't added then... May you could add way to add also non existing worlds... OR something like that

So the main problem is that world couldn't be added on world creation and when I add it for ex. 1 sec later, inventory doesn't change from previous world and it's useless.... and I can't add nether and end variant to the group on player world creation, when the world has to exist... and world is generated just when player go to the end etc.

**How the feature is useful**

This is useful when someone will have this plugin or when you have another similar plugin for creating player worlds.

**Comment by benwoo1110** [comment]
you in theory can do that with the API, but from your explanation, you could just 1. create the world, 2. add them to mv-inv, 3. teleport to the new world

**Comment by bobhenl** [comment]
yeah, but the problem is that nether and end is created when players are teleported there and they don't have then right inventory, cause it's not added in the right time to mvinv. And when I try to add it there after few sec they don't get their inventory back.
